src/databases/indexedDb/loadRecords.ts

```diff
@@ -133,6 +133,28 @@ export async function loadRecords(countUpdater: ((totalCount: number) => void) |
 		return items.slice(start, end);
 	}
 
+	// Handle out-of-line keys (auto-increment with no key path)
+	const primKeySchema = table.schema.primKey;
+	const keyPath = primKeySchema.keyPath;
+	const isOutOfLineKey = (keyPath as string | string[] | null) === null;
```

Comment:
Please update this logic to correctly detect an `outOfLineKey`.

keyPath can be a `string`, `string[]`, or `undefined`, and trimming adds extra safety since some values may contain only whitespace.

```suggestion
	const primKeyPath = table.schema.primKey.keyPath;
	const primaryKey = Array.isArray(primKeyPath)
		? (primKeyPath[0])
		: (primKeyPath);

	const hasOutOfLineKeys = !primaryKey?.trim();
```